__test__/extensions/action-handlers/ngpvan-action.test.js

```diff
@@ -34,7 +34,7 @@ describe("ngpvn-action", () => {
     ).not.toThrowError();
   });
 
-  describe("#getClientChoiceData", async () => {
```

Comment:
I saw a ton of these and left one at random to comment on... is jest now not running `async`?

Reply:
With the new version of Jest, the describe function cannot be defined as an async function. If you add the async keyword, Jest will throw the following error: `Returning a Promise from "describe" is not supported. Tests must be defined synchronously.`

`it`, `beforeEach`, and other Jest functions can still be defined asynchronously, so I think in general Jest is still running partly async. I noticed that we weren't using any instances of `await` in the describe functions anyway, so I was able to remove the `async` keyword from the describe function definitions without having to make related code changes.